content/en/docs/how-to/plugin-management.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Install and manage notation plugins
+description: Install and manage notation plugins
+weight: 9
+---
+
+Plugins for `notation` provide functionality and integration with key stores and signing services. For example:
+- The [notation-azure-kv](https://github.com/Azure/notation-azure-kv) plugin allows you to sign a Notation-generated payload with a private key in [Azure Key Vault](https://learn.microsoft.com/en-us/azure/key-vault/general/overview?wt.mc_id=azurelearn_inproduct_oss_notaryproject).
+- The [AWS Signer plugin for Notation](https://docs.aws.amazon.com/signer/latest/developerguide/image-signing-prerequisites.html) lets you sign and verify images using Notation with keys and certificates managed by [AWS Signer](https://docs.aws.amazon.com/signer/latest/developerguide/Welcome.html), a signing service.
+
+## Install a plugin
+
+To install a plugin, download the plugin binary, create a `{plugin-name}` directory in `{NOTATION_LIBEXEC}/plugins/`, and add the plugin binary to that directory. Alternatively, if you are using an application that bundles a plugin and the `notation` binary together, such as [AWS Signer](https://docs.aws.amazon.com/signer/latest/developerguide/image-signing-prerequisites.html), see the installation instructions from the vendor.
+
+{{% alert title="Warning" color="warning" %}}
```

Comment:
The above is showing as markdown script. Is that intended?

Reply:
My bad. I wasn't using the preview feature when reviewing the content. I was looking at the source code file

Reply:
@zr-msft  - How do I generate/find the Preview URL in the PR ? Like for the above you share the preview link, but I didn't know where to get it on my own.

Reply:
never mind. found it.

Reply:
@iamsamirzon Is the problem still there? If not, we can resolve this comment, then this PR is ready to merge

Reply:
@yizha1 @iamsamirzon i believe the problem is resolved

Reply:
yes it is. This can be merged